lang/python/homeassistant/Makefile

```diff
@@ -0,0 +1,77 @@
+#
+# Copyright (C) 2019 CZ.NIC, z. s. p. o. (https://www.nic.cz/)
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=homeassistant
+PKG_VERSION:=0.98.1
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://files.pythonhosted.org/packages/source/h/homeassistant
+PKG_HASH:=7f8a6a011cf1cf2254a471f0eae1bedaad6dba1dcf3ab67f453d9a80d73ee477
+
+PKG_MAINTAINER:=Josef Schlehofer <pepe.schlehofer@gmail.com>
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE.md
+PKG_CPE_ID:=cpe:/a:home-assistant:home-assistant
+
+include $(INCLUDE_DIR)/package.mk
+include ../python3-package.mk
+
+define Package/$(PKG_NAME)
+  SUBMENU:=Python
+  SECTION:=lang
+  CATEGORY:=Languages
```

Comment:
I would rather put this in utils/Utilities like Domoticz.

Reply:
Ok, I dont have any issues with that.